src/lua/lua_tcp.c

```diff
@@ -342,7 +515,17 @@ lua_tcp_push_error (struct lua_tcp_cbdata *cbd, gboolean is_fatal,
 	struct lua_callback_state cbs;
 	lua_State *L;
 
-	lua_thread_pool_prepare_callback (cbd->task->cfg->lua_thread_pool, &cbs);
+	if (cbd->thread) {
+		va_start (ap, err);
```

Comment:
Why do we need a copy here counting that we don't use the original va_list?

Reply:
fixed